dist.sh

```diff
@@ -14,11 +14,11 @@ done
 
 echo "Creating reference documentation"
 cd ../querydsl-docs
```

Comment:
Also, this is dependent on line 11.
I think t's better to store `pwd` beforehand, and append the modules to that. Since that won't work anymore.